components/next_steps_view/next_steps_view.tsx

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import Card from 'components/card/card';
+import professionalLogo from 'images/cloud-logos/professional.svg';
+
+import './next_steps_view.scss';
+import Accordion from 'components/accordion';
+
+type Props = {
+    skuName: string;
+};
+
+export default class NextStepsView extends React.PureComponent<Props> {
+    getBottomText = () => {
+        // TODO: will be stored in user prefs at a later date
+        const {isFinished} = {isFinished: false};
+
+        if (isFinished) {
+            return (
+                <FormattedMessage
+                    id='next_steps_view.allSetToGo'
+                    defaultMessage={'You\'re all set to go!'}
+                />
+            );
+        }
+
+        return (
+            <FormattedMessage
+                id='next_steps_view.hereAreSomeNextSteps'
+                defaultMessage='Here are some recommended next steps to help you collaborate'
+            />
+        );
+    }
+
+    getLogo = () => {
+        // TODO: Switch logos based on edition once we have the other logos
+
+        switch (this.props.skuName) {
+        default:
+            return professionalLogo;
+        }
+    }
+
+    render() {
+        return (
+            <div
+                id='app-content'
+                className='app__content NextStepsView'
+            >
+                <div className='NextStepsView__header'>
+                    <div className='NextStepsView__header-headerText'>
+                        <div className='NextStepsView__header-headerTopText'>
+                            <FormattedMessage
+                                id='next_steps_view.welcomeToMattermost'
+                                defaultMessage='Welcome to Mattermost'
+                            />
+                        </div>
+                        <div className='NextStepsView__header-headerBottomText'>
+                            {this.getBottomText()}
+                        </div>
+                    </div>
+                    <div className='NextStepsView__header-logo'>
+                        <img src={this.getLogo()}/>
+                    </div>
+                </div>
+                <div className='NextStepsView__body'>
+                    <div className='NextStepsView__body-main'>
+                        <Accordion defaultExpandedKey={'Card_1'}>
+                            {(setExpanded, expandedKey) => {
+                                return (
+                                    <>
```

Comment:
Woah, first I've seen this. Is this essentially a `<React.Fragment>`?

Reply:
Yeah! It's a shorthand they introduced in v16.2.0: https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html